docs/ui/organizations.md

```diff
@@ -81,9 +115,11 @@ Members and service accounts must already be configured for the organization. An
     
     You may set this to "No Access" if you do not want organization members not specifically added to the workspace to access it (organization Admins excepted). 
 
+See the [Workspace sharing](/ui/workspaces/#workspace-sharing) documentation for more information.
+
 ## Organization and workspace roles
 
-Prefect Cloud enables you to configure both organization and workspace roles for users.
+Prefect Cloud enables you to configure both [organization and workspace roles](/ui/roles/) for users.
 
 - Organization roles apply to users across an organization.
 - Workspace roles apply to users within a specific workspace.
```

Comment:
Especially now that we define and differentiate via these bullets, the next line, "Select Roles within an organziation to see the configured roles for your organization" would benefit from additional clarity as well, e.g.:

"Select Roles within an organization to see the configured workspace roles for your organization"